sdk/eventhubs/azure-messaging-eventhubs/vcpkg/vcpkg.json

```diff
@@ -8,19 +8,18 @@
     "Microsoft Azure Messaging Event Hubs SDK for C++",
     "This library provides Azure Messaging Event Hubs SDK."
   ],
-  "homepage": "https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/keyvault/azure-security-keyvault-keys",
+  "homepage": "https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/eventhubs/azure-messaging-eventhubs",
   "license": "MIT",
   "dependencies": [
-    {
-      "name": "azure-core-cpp",
-      "default-features": false,
-      "version>=": "1.9.0"
-    },
```

Comment:
I would have expected to keep this and avoid transitive dependencies. Why not keep this?

Isn't this contributing to the issue:
https://github.com/Azure/azure-sdk-for-cpp/issues/4795

Reply:
That's true, but I am bringing everything to the current way of doing things, and then when fixing another item, will fix the issue everywhere uniformly.